CONTRIBUTING.md

```diff
@@ -0,0 +1,172 @@
+# Contributing to Puppet modules
+
+So you want to contribute to a Puppet module: Great! Below are some instructions
+to get you started doing that very thing while setting expectations around code
+quality as well as a few tips for making the process as easy as possible. 
+
+## Get help
+
+### On the web
+
+* [General GitHub documentation](http://help.github.com/)
+* [GitHub pull request
+  documentation](http://help.github.com/send-pull-requests/)
+
+### On chat
+
+* [Slack](https://slack.puppet.com) in the **#bolt** channel
```

Comment:
I'd maybe make this it's own section right here:
```
## Community Slack Channels

Join the `#bolt` channel in the [Puppet community
Slack](https://slack.puppet.com/) where Bolt developers and community members
who use and contribute to Bolt discuss the tool.
```
If this had more than just tasks and plans in it I'd say we should link to Puppet channels too, but it's really Bolt specific.

Then have this where it was at the bottom:
```
### Github resources

* [General GitHub documentation](http://help.github.com/)
* [GitHub pull request
  documentation](http://help.github.com/send-pull-requests/)
```